doc/specs/staffPage.md

```diff
@@ -0,0 +1,161 @@
+# Staff Page
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+4. Data model
+
+## Motivation
+
+>The staff team is the core of the smooth operation of the site. The staff page is a list of the site's staff members for the user's convenience.
+
+## Business Rules
+
+> * User classes must have an option to be presented on the staff page.
+> * Users with the class management permission must be able to toggle if a user class is displayed on the staff page.
+> * User classes with dedicated staff page descriptions must get their own sections on the staff page (i.e. forum moderators).
+
+## Use Cases
+
+### 1.0 A user visits the staff page
+
+**Primary Actor:** A user
+
+**Brief:** Allows users to see the staff page.
+
+**Preconditions:** N/A
+
+**Trigger:** A user navigates to the staff page.
+
+**Postconditions:** The staff page is rendered.
+
+**Basic Flow:**
+
+1. The user navigates to the staff page.
+2. The staff page is generated, including the username hyperlink and the user's staff page description.
```

Comment:
Additionally, you have several classes which probably should appear in separate sections along with their 'class description'. This is never specified (this would be a functional req).